client/react_app/src/components/GenericButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {Button} from 'react-bootstrap';
+
+interface Props {
+  onClick: React.MouseEventHandler<HTMLElement> | undefined,
```

Comment:
onClick ma być opcjonalnym propem? Jeśli tak to można użyć tego https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties

Reply:
\+ w interfejsach używa się raczej `;` zamiast `,` na końcu linijki

Reply:
done